_data/users.yml

```diff
@@ -417,6 +417,20 @@
   links:
     - urltext: View the Hudson River Trading website
       url: https://www.hudsonrivertrading.com/
+- name: HugoBank
+  highlight: false
+  anchor: hugobank
+  logo: /assets/images/logos/hugobank.png
+  logosmall: /assets/images/logos/hugobank-small.png
+  testimonial: |
+    HugoBank uses the power of Trino for secure, efficient, and optimized
```

Comment:
This is SUPER generic .. can we maybe at least indicate what their Trino-based data platform is for and reword a bit along that .. 

e.g. customer financial data? internal analytics or data access for customers or what? Also can be indicating if they have a lakehouse or whatever .. you should find info in the recording and the slide deck

Reply:
yes. will edit. Pulled initially from their session abstract, but let me update via slides.